src/App.jsx

```diff
@@ -1,33 +1,58 @@
 import './App.scss';
+import { useState } from 'react';
+
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+function getFilteredMovies(movies, query) {
+  let preparedMovies = [...movies];
+
+  if (query) {
+    const normalizedQuery = query.toLowerCase().trim();
+
+    preparedMovies = preparedMovies.filter(movie => (
+      movie.title.toLowerCase().includes(normalizedQuery)
+      || movie.description.toLowerCase().includes(normalizedQuery)
+    ));
+  }
+
+  return preparedMovies;
+}
```

Comment:
This code looks good, but if you invert the check you will make it better

Also you can use destructuring for `movie`

```suggestion
function getFilteredMovies(movies, query) {
  if (!query) {
    return movies;
  }

  const normalizedQuery = query.toLowerCase().trim();

  return movies.filter(
    movie =>
      movie.title.toLowerCase().includes(normalizedQuery) ||
      movie.description.toLowerCase().includes(normalizedQuery),
  );
}
```